verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -317,6 +366,79 @@ std::vector<verible::lsp::Location> SymbolTableHandler::FindReferencesLocations(
   return locations;
 }
 
+std::optional<verible::lsp::Range>
+SymbolTableHandler::FindRenameableRangeAtCursor(
+    const verible::lsp::PrepareRenameParams &params,
+    const verilog::BufferTrackerContainer &parsed_buffers) {
+  Prepare();
+  if (files_dirty_) {
+    BuildProjectSymbolTable();
+  }
+  std::optional<verible::TokenInfo> symbol =
+      GetTokenInfoAtTextDocumentPosition(params, parsed_buffers);
+  if (symbol.has_value()) {
```

Comment:
```suggestion
  if (symbol) {
```
for consistency, as this is how you use std::optional below (`if (!location)`)